Color revision

Astakhin opened the pull request:
Due to problems with my github account, I'm reopening this PR: #13969 

New colors: *Screenshots in #13969*

Unlike the previous PR, I decided to keep some things in their current form (a lot of color variables)

Disabled button has been fixed:

Before: (Why it's gray if it's already blurred?)

![{F2FC6EB5-D8E4-40C6-AA83-AE086AF96A33}](https://github.com/user-attachments/assets/fbc4d09d-c0cd-461f-8891-7f485d412032)

After:

![{726A7D3B-6745-461D-8D70-1246EDE2899B}](https://github.com/user-attachments/assets/4c81c8e3-fde3-4399-ab23-6cd018d1dd72)

The crooked logo has been fixed in all files:

Before: (If this was a conscious author's idea, let me know)

![image](https://github.com/user-attachments/assets/c1e53b9c-ed24-42ce-97be-7ec448338f95)

After:

![image](https://github.com/user-attachments/assets/a70e3c6d-d837-4bb2-9311-dae40aeff8ce)

New icon (including MacOS):

![{4093AEBB-FC70-4010-9690-0EC27AB304ED}](https://github.com/user-attachments/assets/1891b973-c480-4de0-9d88-168861a0d848)


If this PR will be approved, I will create another PR with all the updated assets including:
SVG / PNG logos, some stuff from [ZIPs](https://github.com/zkSNACKs/ZIPs) and [WasabiSwag](https://github.com/zkSNACKs/WasabiSwag) (if it's still relevant)

Astakhin commented:
Can someone review this PR?

lontivero commented:
@turbolay any update?

Kruwed reviewed (changes requested):
Overall, I think this PR is an improvement. The white text inside the buttons fits in much better, especially on light mode.

As for the new green color, it's easy to get used to and seems more aesthetic. For logo branding, I don't know whether a pale/subtle color scheme or a bright/obnoxious one is more effective for capturing professional interest at a first glance.

Keeping the Send/Receive/Donate buttons green instead of turning them to grey whenever a menu is opened is a good change. Other colors don't fade, so these buttons should behave the same way. However, the "Donate" button is still getting greyed out, so it should also stay green.

The only thing I don't like is the glare/gradient for the tray icon, I would prefer solid black.

turbolay commented:
Sorry to take time on this PR.
I've tested a bit. I think it's OK.

I've noticed a couple of bugs (like the welcome page `oobe = true` on light theme).

There is a need to re-introduce a difference between Next buttons (button that induces a next step in the navigation) and action buttons (like Wallet Settings -> Tools -> Open buttons). The color used on the action buttons should be different., Maybe greyish? IDK

Astakhin commented:
New animation visualizes the coinjoin round and the download at the same time (the client expects action from the user)

And less code

![{384F27D8-3C37-4681-B0F7-A4529157038D}](https://github.com/user-attachments/assets/abf0e428-7cb6-45ec-9f2a-7da9e14aa07c)

![{96F88009-B787-41A7-A4CF-3EF1004519F3}](https://github.com/user-attachments/assets/26f4bfe1-604f-437c-aea3-a5a5d2b49623)

Astakhin commented:
> There is a need to re-introduce a difference between Next buttons (button that induces a next step in the navigation) and action buttons (like Wallet Settings -> Tools -> Open buttons). The color used on the action buttons should be different., Maybe greyish? IDK

![{8514E844-A121-4B7E-98FD-FCD1B510D718}](https://github.com/user-attachments/assets/57f1dd5f-3d93-4e75-98fa-46077b2628af)

![{3FA0ED43-0150-4134-9C63-F0F536C060AF}](https://github.com/user-attachments/assets/5d45efaa-47f6-47ba-a79f-9afdc036e4fc)

New grey action buttons. Including inactive, active, pointer over, pressed for dark and light themes.

MarnixCroes reviewed (commented):
There is a visual issue when a button is supposed to be grayed out:

### master
![image](https://github.com/user-attachments/assets/38b3ebe9-faed-47cb-ac8e-659def959dfc)

### PR
![Screenshot from 2025-06-18 19-40-34](https://github.com/user-attachments/assets/ee20c4c9-a5bb-4298-860c-fe83f1fe067c)

---
the main taskbar and toolbar icon differ, they're supposed to be the same?

![Screenshot from 2025-06-18 17-39-12](https://github.com/user-attachments/assets/1de1b92b-e524-40e1-ba98-9d40f3aac54e)

![Screenshot from 2025-06-18 17-39-32](https://github.com/user-attachments/assets/e1244712-454d-45b5-9745-a793c04b0f5a)

Astakhin commented:
> There is a visual issue when a button is supposed to be grayed out:
Thanks for noticing. Fixed.

> the main taskbar and toolbar icon differ, they're supposed to be the same?

It's OS caching of the icons, I replaced all of the icons in first commit, there is no more old icons

Astakhin commented:
Help me someone. There is one thing that i can not fix for 2 days:

![image](https://github.com/user-attachments/assets/34f7bd3a-26eb-4166-b0c3-30c3cb43b3d9)

Kruwed reviewed (changes requested):
The taskbar and toolbar icons are still different from each other when I test, not sure how to get them to match

> Help me someone. There is one thing that i can not fix for 2 days:

This is the PR that introduced the rename wallet feature, not sure if it's helpful - https://github.com/WalletWasabi/WalletWasabi/pull/11652

lontivero commented on WalletWasabi.Fluent/Views/Settings/SettingsPageView.axaml:
```diff
@@ -14,7 +14,7 @@
                  EnableNext="True"
                  EnableCancel="{Binding !IsReadOnly}"
-                 NextContent="Done"
+                 NextContent="Save"
```
Adding more unrelated changes won't make it easier to merge, instead it can create more controversy and distractions.

Astakhin commented:
To summarize:

> I've noticed a couple of bugs (like the welcome page `oobe = true` on light theme).

If you mean an incorrect display of the logo in the welcome page on light theme, then i fixed it

> There is a need to re-introduce a difference between Next buttons (button that induces a next step in the navigation) and action buttons (like Wallet Settings -> Tools -> Open buttons). The color used on the action buttons should be different., Maybe greyish? IDK

I made the action buttons gray. I think it looks good. Waiting for your review

> There is a visual issue when a button is supposed to be grayed out:

I fixed it (that was too difficult because it needs to make changes in interactions between mainscreen and windows)

> the main taskbar and toolbar icon differ, they're supposed to be the same?
> The taskbar and toolbar icons are still different from each other when I test, not sure how to get them to match

I fixed it. Thank you for noticing. Difference between taskbar and tool bar may be related to OS icon caching.

> This is the PR that introduced the rename wallet feature, not sure if it's helpful - #11652

Special thanks for this advice. I was able to fix the problem of incorrectly displaying the colors of the 3rd-Layer window (Rename Wallet).

To fix this, I moved the function of Rename Wallet from WalletRenameView to WalletSettingsGeneralView (with WalletRename ViewModels)

Now it looks:

![{AF834E05-2227-4203-8C92-5F26EA16528E}](https://github.com/user-attachments/assets/f0c0554c-cbc6-4586-8247-e0054ac10025)

MarnixCroes reviewed (commented):
thank you for your dedication, some comments:

2f0565a4ae7206fa406ae29b2bdf60041119fe02

- The selected coinjoin strategy is hard to notice
I can see which one, but a user that doesn't know that you select one it's not easy imo.
### PR
![Screenshot from 2025-06-23 10-12-54](https://github.com/user-attachments/assets/183ebcaf-b912-4bff-931b-0767ef13591b)

### master
![image](https://github.com/user-attachments/assets/6ce43e21-cd42-4696-82ec-691b69edcc71)

- I'm not sure if some buttons being gray now instead of green is an improvement tbh
### PR
![Screenshot from 2025-06-23 10-13-40](https://github.com/user-attachments/assets/80560707-c73e-4bab-8e44-598c01e15e8a)
### master
![Screenshot from 2025-06-23 10-14-20](https://github.com/user-attachments/assets/ca86afb3-f7cc-4338-b0a5-327cb48ae517)

### PR
![Screenshot from 2025-06-23 10-35-15](https://github.com/user-attachments/assets/8e74419c-f71c-49d9-b8b4-1bc374cfe6d7)

### master
![Screenshot from 2025-06-23 10-25-58](https://github.com/user-attachments/assets/dbe2d83f-a297-4a06-bfc2-4168155dab55)

turbolay reviewed (approved):
There is no doubt the result is better than current Wasabi.
The light theme is still really ugly. If you have ideas and want to improve it feel free.
This PR damages a bit Wasabi's brand identity but I think it's ok, we can adapt.
